src/main/java/io/github/tryexceptelse/jdex/Settings.java

```diff
@@ -3,8 +3,4 @@
 /**
  * Settings object to store program settings
  */
-class Settings {
-    String autoSaveLoc;
-    String defaultSaveLoc;
-    String lastSavedFile;
-}
+class Settings {} // skeleton
```

Comment:
It may end up with methods to load settings, but other than that, it's only storing settings data